README.md

```diff
@@ -1,8 +1,46 @@
-# job_tracker
+# UpWiSE
 
-This is a WIP open source project to meet the following objectives:
+**UpWiSE** is an open-source project designed to help users efficiently track and organize their job applications. The app allows users to monitor their application progress, store notes, and attach relevant job links. It is created to simplify job hunting by consolidating all application details in one place.
```

Comment:
This section needs to be updated to have the project name be "Job Tracker", and the group that is sponsoring this project is called UpWiSE (upwisesc.org).